src/components/Display.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Display.scss';
+
+const Display = ({ result, onChange }) => {
+  const handleChange = e => {
+    if (e.target.value === '') {
+      onChange('0');
+    } else {
+      onChange(e.target.value);
+    }
```

Comment:
This looks like a good case for the ternary operator, or even just check if there is a value or not:

````js
onChange(e.target.value === '' ? '0' : e.target.value);
// or maybe even: onChange(e.target.value || '0');
```